### DISCUSSION TITLE
Question about kanban view

### DISCUSSION THREAD

**Discussion by Himlol** [mabeyj/trilium-collection-views#25] [Ideas]
Hi,

You made amazing extension, i'm very exciting to use it. I would like to ask, are you planning add drag&drop feature to board view with set groupBy label after drop? 
I tried for several days to implement this on my own, but I did not succeed. I think this feature would make a kanban board in trilium complete.

**Comment by mabeyj** [comment]
Hi, thanks! Yes, drag and drop is planned. There is an open feature request for it here: https://github.com/mabeyj/trilium-collection-views/issues/16
